src/parsec_service/build_run.md

```diff
@@ -94,4 +94,18 @@ The TPM provider will try to build the `tss-esapi` crate which needs built TSS 2
 libraries. It will use `pkg-config` to find them using the names `tss2-esys` and `tss2-tctildr`.
 Make sure you also follow the requirements of the [tss-esapi crate](https://docs.rs/tss-esapi).
 
+### CryptoAuth Library
+
+TODO
```

Comment:
Maybe `Work in progress...` or smth like that? Though `TODO` is alright too

Reply:
Yes `TODO` can easily be grepped 😬